gateway-service-baseline-lib/build.gradle.kts

```diff
@@ -13,14 +13,13 @@ dependencies {
   api(project(":gateway-service-api"))
 
   // Common utilities
-  implementation("org.apache.commons:commons-lang3:3.10")
-  implementation("org.apache.commons:commons-math:2.2")
-  implementation("com.google.protobuf:protobuf-java-util:3.17.3")
+  implementation("org.apache.commons:commons-lang3:3.12.0")
+  implementation("org.apache.commons:commons-math3:3.6.1")
```

Comment:
Considering math3 replaces the v2 line in 2011 or 2012 I'm not sure how we ended up here in the first place, but I'm going to go ahead and upgrade...